report/sections/Chapter 1/Intro.tex

```diff
@@ -1 +1,72 @@
-\chapter{P7 Intro}
+\chapter{Introduction} \label{chap:introduction}
+On the 1st semester of the Software Master’s Degree at \aau{}, the Programming Paradigms course had a problem where most attending students failed the course last year. 
+This resulted in a survey being sent out, and a student task force being assembled to grasp the main flaws with how the course was structured. 
+Here, many students mentioned lacking experience in the programming languages taught in the course given few opportunities to actively train their skills during the semester. 
+
+
+With this in mind, \aau{} chose to restructure the course entirely. 
+The course now focuses on a single programming language instead of multiple. 
+Furthermore, the course now features more practical programming exercises with classroom discussion in-between, rather than traditional lectures followed by a seperate exercise session.
+To support this way of teaching, the course lectures have been moved to a more classical classroom setup, but with a monitor at each table and a main monitor at the lecture desk. 
+This is meant to allow students to more easily work together in groups by using the shared monitor.
+In addition, it allows the professor to view and share other groups' monitors such that student solutions can be used for classroom discussions. 
+
+
+However, with this way of teaching new problems arise.
+Since students use their own local development environments for solving exercises, other students cannot revisit the discussed exercise solutions as these are not shared, making it impossible to revisit them later. 
```

Comment:
```suggestion
Since students use their own local development environments for programming exercises, other students cannot revisit the discussed exercise solutions created by other groups. This makes it difficult to reflect on and learn from other students' solutions.
```